public/app/screens/main/sections/settings/user-info/style.scss

```diff
@@ -0,0 +1,26 @@
+@import "theme";
+
+.cc-user-info {
+  .cc-user-info-item {
+    @include grid-row();
+
+    margin: convert-to-rem(20px) 0;
+
+    &__label-container {
+      @include grid-column($columns: 10, $float: left);
+
+      .cc-user-info-item__label {
```

Comment:
If you cannot nest the class names using `&` that's a sign for net good enough naming